src/constants/icons.tsx

```diff
@@ -20,49 +21,49 @@ import Sherlock from "public/assets/icons/projects/sherlock.svg";
 import Rated from "public/assets/icons/projects/rated.svg";
 import Unknown from "public/assets/icons/projects/unknown.svg";
 
-export const projectIcons: Record<string, JSX.Element> = {
-  Unknown: <Unknown />,
-  "Cozy Finance": <Cozy />,
-  Polymarket: <Polymarket />,
-  Across: <Across />,
-  Sherlock: <Sherlock />,
-  OSnap: <OSnap />,
-  Rated: <Rated />,
+export const projectIcons = {
```

Comment:
typescript and nextjs have updated how they handle these svg as react components so that we don't need to type them like this anymore. this is great because it lets us use these svgs directly as components and pass classes to them